composer.json

```diff
@@ -22,17 +22,23 @@
         "issues": "http://github.com/liip/RMT/issues"
     },
     "require": {
-        "php": ">=5.3.9",
+        "php": "^5.6|^7.0",
```

Comment:
No we should allow 7.0 and 7.1 (and 7.x). We currently test php 7.0 and 7.1 so each supported version is covered